src/enclave_proc/commands.rs

```diff
@@ -48,12 +48,24 @@ pub fn run_enclaves(
         .map_err(|e| e.add_subaction("Failed to construct CPU information".to_string()))?
         .get_cpu_config(args)
         .map_err(|e| e.add_subaction("Failed to get CPU configuration".to_string()))?;
+
+    let enclave_name = match args.enclave_name.clone() {
```

Comment:
Is cloning necessary?

Reply:
could use to_string later and borrowing here